go/oasis-node/cmd/registry/node/node.go

```diff
@@ -352,7 +359,7 @@ func Register(parentCmd *cobra.Command) {
 func init() {
 	flags.String(CfgEntityID, "", "Entity ID that controls this node")
 	flags.Uint64(CfgExpiration, 0, "Epoch that the node registration should expire")
-	flags.StringSlice(CfgCommitteeAddress, nil, "Address(es) the node can be reached as a committee member")
+	flags.StringSlice(CfgCommitteeAddress, nil, "Address(es) the node can be reached as a committee member of the form [Certificate@]ip:port (where Certificate@ part is optional and represents base64 encoded node certificate for TLS)")
```

Comment:
yeah indeed. So for this exact command the annoyance can be a bit avoided by instead accepting the certificate path here. But yeah the general problem remains, and as you mentioned, we can only avoid the need for storing the certificates in the registry by rewriting the TLS authentication. Can open an issue for this in case you think we might want do to it.

Reply:
> Can open an issue for this in case you think we might want do to it sooner or later.

Yeah I think this could be useful to have, so file an issue and we'll see when/if we can do it.

Reply:
#2556